tamr_unify_client/models/attribute_configuration/collection.py

```diff
@@ -0,0 +1,77 @@
+from tamr_unify_client.models.attribute_configuration.resource import (
+    AttributeConfiguration,
+)
+from tamr_unify_client.models.base_collection import BaseCollection
+
+
+class AttributeConfigurationCollection(BaseCollection):
+    """Collection of :class'tamr_unify_client.models.attribute_configuration.resource.AttributeConfigurations
+    :param client: Client for API call delegation.
+    :type client: :class:`~tamr_unify_client.Client`
+    :param data: JSON data representing this resource
+    :type data: dict
+    :param api_path: API path used to access this collection.
+        E.g. ``"projects/1/attributeConfigurations"``.
+    :type api_path: str
+    """
```

Comment:
These seem like the parameters for`__init__`, but `__init__` is handled by the superclass

Reply:
attribute's collection.py and project's collection.py have these, so I assumed it was tamr documentation/syntax

Reply:
Oh weird looks like the collections all have them but resources don't